lms/templates/footer-edx-new.html

```diff
@@ -30,17 +30,17 @@ <h2 class="sr footer-about-title">${_("About edX")}</h2>
       </div>
 
       <div class="footer-about-copyright">
-	## Using "edX Inc." explicitly here for copyright purposes (settings.PLATFORM_NAME is just "edX", and this footer is only used on edx.org)
-	<p>&copy; ${settings.COPYRIGHT_YEAR} edX Inc.</p>
+      	## Using "edX Inc." explicitly here for copyright purposes (settings.PLATFORM_NAME is just "edX", and this footer is only used on edx.org)
```

Comment:
NOTE: also, just pushed a quick fixup/squash-able commit to indent the HTML here properly.